Sources/Authgear.swift

```diff
@@ -709,16 +709,6 @@ public class Authgear {
                 return handler(.failure(wrapError(error: error)))
             }
         }
-        if case let .failure(error) = Result(catching: { try sharedStorage.delIDToken(namespace: name) }) {
-            if !force {
-                return handler(.failure(wrapError(error: error)))
-            }
-        }
-        if case let .failure(error) = Result(catching: { try sharedStorage.delDeviceSecret(namespace: name) }) {
-            if !force {
-                return handler(.failure(wrapError(error: error)))
-            }
-        }
```

Comment:
But in js SDK, we clear these https://github.com/authgear/authgear-sdk-js/commit/d3e192ba56d5720011ca021bfca301c9a794ecc2#diff-b4cf13ab6d28203079e35151e4a0305be21c3c310d82a76c51530de127372bbaR250

Reply:
That should be a mistake in js sdk, we should not clear the saved device secret in the shared storage (Because it could be generated by another app, if the storage is really shared). I will fix it in the js sdk.

Reply:
Per offline discussion, we will still clear them on logout.